jha/src/views/enrolment/ContactInfoPage.vue

```diff
@@ -139,20 +139,34 @@
             </div>
           </div>
         </div>
-        <input type="checkbox" id="same-address-check" v-model="isMailSame">
-        <label for="same-address-check">This is my mailing address.</label>
+        <div v-if="!onlyFPC">
+          <input type="checkbox" id="same-address-check" v-model="isMailSame">
+          <label for="same-address-check">This is my mailing address.</label>
```

Comment:
Now that we have the "Checkbox" component in the common library, can you use that instead?

Reply:
changed